core/jazz_acl/components/scm/bitbucket.js

```diff
@@ -0,0 +1,91 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const request = require("request");
+const logger = require("../logger.js");
+
+const addRepoPermissionInBitbucket = async (config, serviceInfo, policies) => {
+  let users_list = [];
+
+  const addUserPermsInBitbucketRepo = async (count) => {
+    try {
+      let policy = policies.details[count];
+      const repoInfo = {
+        "repo_name": `${serviceInfo.domain}_${serviceInfo.service}`,
```

Comment:
Don't we have the repository details in service metadata (you have `serviceInfo` available already)? Should we construct it here?

Reply:
serviceInfo have the repository url.  Here repo name is needed.